src/commands/deployments/editScmType.ts

```diff
@@ -18,9 +18,11 @@ export async function editScmType(context: IActionContext, node?: SiteTreeItem |
         node = <SiteTreeItem>node.parent;
     }
 
-    await appservice.editScmType(node.root.client, node, context, newScmType, showToast);
-
-    if (node.deploymentsNode) {
-        await node.deploymentsNode.refresh();
+    if (node instanceof SiteTreeItem) {
```

Comment:
I'm not sure we need this check.  You could have an else and throw the not supported error, but honestly we have the typings here to prevent people passing in the wrong types of treeItems so I think it's more on the coder to ensure that they pass in the right treeItem rather than throwing an error here.